introduction/what-is.md

```diff
@@ -0,0 +1,48 @@
+This file holds the text to accompany the [What is InnerSource?](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321608) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# What is InnerSource?
+
+Let's say that team A uses software produced by team B.
+Team A submits a feature request to team B, but team B isn't able to implement that feature in time for team A.
+In an InnerSource setting, if team A can't get this feature request then it submits a pull request instead.
+That is to say team A implements the feature directly in team B's software and submits a pull request with the code changes.
+Team B partners to review and accept the submitted code.
+
+In this example, we call team A the **Guest** team and team B the **Host** team.
+The terms **Guest** and **Host** suggest a situation analogous to receiving a visitor in the home.
+In that situation, most people want to be a good host.
+They ensure that things are kept neat and tidy in anticipation of their guests' arrival.
+Visitors are greeted at the door and invited in.
+They can use the features and utilities that are in the home's public areas.
+There may be a few house rules that guests are asked to follow.
+Similarly, most guests want to show respect for the home and their host.
+They're careful with the items in the house and follow the rules for the duration of their stay.
+They may hope for or expect a return invitation as long as they've been courteous and polite.
+These concepts around a home visit are a metaphor for the attitude and behaviors that teams should bring as one hosts another making a guest contribution to the codebase.
+
+Let's take a closer look at how the mechanics of the inner source process can work.
+To aid in this explanation, we'll name a few key individuals on the guest and host teams.
+First, the **Product Owner** determines what functionality the host team is willing to accept as a contribution.
```

Comment:
- Danese mentions an interaction of both teams' POs in the video if I remember correctly. I agree with what is written here, but there's a bit of an ambiguity here.
- I was wondering if it would be a good idea to mention that PO and TC can be same person. This is likely not that uncommon with the usual grass-roots style emerging local library/tool/automation/... projects. Other more "planned" projects might have dedicated PO's from the start, however this does not have to be the case - matching superior topic: governance.

Reply:
This subject keeps coming up - updated.

Reply:
:+1: